### PR TITLE
Addressing Suggestions in Pull Requests #14 and #16

### PR DESCRIPTION
**Pull request by amanial1234**
- Deleted Unused User files
- Removed White Space in numerous files
- Improved formatting in numerous files
- Deleted unused delegate in EditViewController.h file
- Correct CamelCase variable name MatchmakingViewController

TESTING

TestCase 1: Used the build simulator and tested the Edit Profile feature to make sure worked without User

https://user-images.githubusercontent.com/103143506/182939781-e8ffe928-3b8e-47d7-81e9-4c85cadb088b.mov

### CODE REVIEW

File: MusicTaste/MusicTaste/View Controllers/Login/ConnectViewController.m [hunk 1]
```diff
@@ -26,7 +24,7 @@ - (void)didTapAuthButton:(ConnectButton *)sender
         if (!error) {
             //Once authorize the ConnectViewController will segue to the LoginView Controller
             [self performSegueWithIdentifier:@"tabBarSegue" sender:nil];
-        }else{}
```
Comment (amanial1234) [inline MusicTaste/MusicTaste/View Controllers/Login/ConnectViewController.m:29]: deleted unnecessary else statement

File: MusicTaste/MusicTaste/View Controllers/Login/ConnectViewController.m [hunk 2]
```diff
@@ -16,8 +16,6 @@ - (void)viewDidLoad
     [super viewDidLoad];
 }
 #pragma mark - Actions
--(void)viewDidAppear:(BOOL)animated{
```
Comment (amanial1234) [inline MusicTaste/MusicTaste/View Controllers/Login/ConnectViewController.m:19]: Deleted unused function

File: MusicTaste/MusicTaste/View Controllers/MatchmakingViewController.m [hunk 3]
```diff
@@ -97,9 +96,9 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
     //Segue to Details View Controller
     if([[segue identifier] isEqualToString:@"detailsSegue"]) {
         UITableViewCell *cell = sender;
-        NSIndexPath *indexpath = [self.MatchTableView indexPathForCell:cell];
+        NSIndexPath *indexPath = [self.MatchTableView indexPathForCell:cell];
```
Comment (amanial1234) [inline MusicTaste/MusicTaste/View Controllers/MatchmakingViewController.m:99]: CamelCase indexpath to indexPath

File: MusicTaste/MusicTaste/APIManagers/SpotifyAPIManager.h [hunk 4]
```diff
@@ -13,7 +12,6 @@ NS_ASSUME_NONNULL_END
 @property (nonatomic, strong) SPTSessionManager *sessionManager;
 @property (nonatomic, strong) SPTConfiguration *configuration;
 @property (nonatomic, strong) NSString *token;
-@property (nonatomic, strong) User *user;// Contains Users author's name, screenname, and profile image.
```
Comment (amanial1234) [inline MusicTaste/MusicTaste/APIManagers/SpotifyAPIManager.h:16]: Deleted User Variable

File: MusicTaste/MusicTaste/APIManagers/SpotifyAPIManager.m [hunk 5]
```diff
@@ -20,7 +18,6 @@ + (instancetype)shared {
     return sharedManager;
 }
 
-
```
Comment (amanial1234) [inline MusicTaste/MusicTaste/APIManagers/SpotifyAPIManager.m:23]: Deleted White Space

File: MusicTaste/MusicTaste/APIManagers/SpotifyAPIManager.m [hunk 6]
```diff
@@ -171,14 +167,13 @@ -(void) saveSpotifyData:(NSDictionary *) spotifyData{
                     NSArray *images = userDict[@"images"];
                     NSDictionary *imageDict = [images objectAtIndex:0];
                     if (imageDict != nil){
-                        NSString * url =[NSString stringWithFormat:imageDict[@"url"], nil];
+                        NSString * url = [NSString stringWithFormat:imageDict[@"url"], nil];
```
Comment (amanial1234) [inline MusicTaste/MusicTaste/APIManagers/SpotifyAPIManager.m:170]: Corrected formatting

File: MusicTaste/MusicTaste/APIManagers/SpotifyAPIManager.m [hunk 7]
```diff
@@ -171,14 +167,13 @@ -(void) saveSpotifyData:(NSDictionary *) spotifyData{
...
                         music[@"username"] = [NSString stringWithFormat: name, nil];
                         music[@"userimage"] = url;
                         [music saveEventually];
                     }
                 }
             }];
-            [music saveEventually];
```
Comment (amanial1234) [inline MusicTaste/MusicTaste/APIManagers/SpotifyAPIManager.m:181]: deleted repetitive call to save Music Query

File: MusicTaste/MusicTaste/View Controllers/ProfileViewController.m [hunk 8]
```diff
@@ -86,7 +85,6 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
     if([[segue identifier] isEqualToString:@"editSegue"]) {
         UINavigationController *navigationController = [segue destinationViewController];
         EditViewController *editController = (EditViewController*)navigationController.topViewController;
-        editController.delegate = self;
```
Comment (amanial1234) [inline MusicTaste/MusicTaste/View Controllers/ProfileViewController.m:89]: Deleted unused delegate of EditViewController

File: MusicTaste/MusicTaste/View Controllers/EditViewController.h [hunk 9]
```diff
@@ -2,13 +2,9 @@
 #import "Parse/Parse.h"
 
 NS_ASSUME_NONNULL_BEGIN
-@protocol EditViewControllerDelegate
```
Comment (amanial1234) [inline MusicTaste/MusicTaste/View Controllers/EditViewController.h:5]: Deleted unused Protocol

File: MusicTaste/MusicTaste/Cells/User.h [hunk 10]
```diff
@@ -1,18 +0,0 @@
-#import <Foundation/Foundation.h>
```
Comment (amanial1234) [inline MusicTaste/MusicTaste/Cells/User.h:1]: Deleted Both User Files

**Review by amanial1234 — COMMENTED** [review]
Cleaned up my code to address the recent pull request comments! Thank you for pointing out the errors!

**Review by nathanielcook — COMMENTED** [review]
Nice!

**Pull request merged**